armv7 binaries are broken: random number generator:RAND_DRBG_instantiate:error retrieving entropy

Crypto functionality is broken in the static armv7 node binaries (tested Node v12, v14 and v16 from this repo's releases). Example:

```
$ curl -LO https://github.com/maxb2/static-node-binaries/releases/download/v12.22.3/node-static-v12.22.3-linux-armv7l
$ chmod +x node-static-v12.22.3-linux-armv7l

$ docker run -it --platform linux/arm/v7 -v ${PWD}:/mnt/Downloads debian:10 /mnt/Downloads/node-static-v12.22.3-linux-armv7l -pe 'crypto.randomBytes(8)'
internal/crypto/random.js:202
  if (ex) throw ex;
          ^

Error: error:2406C06E:random number generator:RAND_DRBG_instantiate:error retrieving entropy
    at Object.randomBytes (internal/crypto/random.js:59:37)
    at [eval]:1:8
    at Script.runInThisContext (vm.js:120:18)
    at Object.runInThisContext (vm.js:309:38)
    at Object.<anonymous> ([eval]-wrapper:10:26)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at evalScript (internal/process/execution.js:94:25)
    at internal/main/eval_string.js:23:3 {
  opensslErrorStack: [
    'error:2406B072:random number generator:RAND_DRBG_generate:in error state',
    'error:2406C06E:random number generator:RAND_DRBG_instantiate:error retrieving entropy',
    'error:2406B072:random number generator:RAND_DRBG_generate:in error state',
    'error:2406C06E:random number generator:RAND_DRBG_instantiate:error retrieving entropy'
  ]
}
```

Compare with the expected output of a dynamically compiled Node.js executable:
```
$ docker run -it --platform linux/arm/v7 arm32v7/node:12 node -pe 'crypto.randomBytes(8)'
<Buffer a6 0c 6a c3 18 dd da ea>
```

Replacing `debian:10` with `alpine:3.11.3` or `alpine:3.14.0` in the `docker run` command line above does not make a difference. I also do **not** believe that it is related to the version of the Alpine base image (e.g. 3.14.0 vs 3.11.3) in this repo's [Dockerfile](https://github.com/maxb2/static-node-binaries/blob/main/Dockerfile).

Maybe some crypto library, say openssl, was missing during the compilation of Node.js, although it would be strange that it only applies to armv7 and not arm64. We should dig in the compilation messages for any hints. The fix might be as simple as an adding some package to the `apk add` command line. Or we may find the issue is really hard to solve and we cannot use Alpine -- I hope it's not the case.

Note that crypto is not just for "cryptography apps" (whatever that means!). For example, `caxa` itself uses it, in fact that's how I first came across the error: https://github.com/leafac/caxa/blob/v2.1.0/src/index.ts#L43

I added a [test stage](https://github.com/maxb2/static-node-binaries/blob/main/Dockerfile#L18) to the Dockerfile so we can confirm the builds in the future. FYI, I kept everything in separate stages so we can easily build to a targeted stage and then tinker. 

The armv7 build fails `make test` with a segfault and a completely unhelpful error message:
```
make[1]: *** [Makefile:698: tools/doc/node_modules] Segmentation fault (core dumped)
make: *** [Makefile:317: test-only] Error 2
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -9,7 +9,7 @@ RUN git clone --depth 1 https://github.com/nodejs/node --branch ${NODE_VERSION} 
 
 FROM fetch as configure
 WORKDIR /usr/src/app/node
-RUN ./configure --fully-static --enable-static
+RUN if [ $(uname -m) = armv7l ]; then ./configure --fully-static --enable-static --openssl-no-asm; else ./configure --fully-static; fi
 
 FROM configure as compile
 ARG NPROCS=1

--- a/Dockerfile
+++ b/Dockerfile
@@ -9,7 +9,7 @@ RUN git clone --depth 1 https://github.com/nodejs/node --branch ${NODE_VERSION} 
 
 FROM fetch as configure
 WORKDIR /usr/src/app/node
-RUN if [ $(uname -m) = armv7l ]; then ./configure --fully-static --enable-static --openssl-no-asm; else ./configure --fully-static; fi
+RUN if [ $(uname -m) = armv7l ]; then ./configure --fully-static --enable-static --openssl-no-asm; else ./configure --fully-static --enable-static; fi
 
 FROM configure as compile
 ARG NPROCS=1

--- a/Dockerfile
+++ b/Dockerfile
@@ -9,7 +9,7 @@ RUN git clone --depth 1 https://github.com/nodejs/node --branch ${NODE_VERSION} 
 
 FROM fetch as configure
 WORKDIR /usr/src/app/node
-RUN if [ $(uname -m) = armv7l ]; then ./configure --fully-static --enable-static --openssl-no-asm; else ./configure --fully-static --enable-static; fi
+RUN ./configure --fully-static --enable-static $([ $(uname -m) = armv7l ] && echo --openssl-no-asm)
 
 FROM configure as compile
 ARG NPROCS=1

